pkg/utils/utils.go

```diff
@@ -372,6 +373,8 @@ func Cleanup() (err error) {
 		}
 		vol := Vol
 		TransitionVolList[vol.Spec.Volume.Name] = apis.CStorVolumeAttachmentStatusUnmountUnderProgress
+
+		logrus.Infof("Volume: %v during cleanup is in %v state", vol.Spec.Volume.Name, TransitionVolList[vol.Spec.Volume.Name])
```

Comment:
How about below line?
```suggestion
		logrus.Infof("Volume: %v marked as %v as part of cleanup activity", vol.Spec.Volume.Name, TransitionVolList[vol.Spec.Volume.Name])
```